core/lib/dal/migrations/20231009073918_drop-prover_jobs_fri_l1_batch_number_fkey.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE prover_jobs_fri ADD CONSTRAINT prover_jobs_fri_l1_batch_number_fkey
+    FOREIGN KEY (l1_batch_number) REFERENCES l1_batches (number);
```

Comment:
The original constraint also has `ON DELETE CASCADE` but I don't think it's a big deal.